src/around.js

```diff
@@ -0,0 +1,34 @@
+var curryN = require('./curryN');
+/**
+ * Accepts three functions, `fore`, `hind` and `fn`, and a single value.
+ * `R.around` effectively wraps `fn` with `fore` and `hind` by first passing the
+ * given value to `fore`, which returns a value that is then passed to `fn`,
+ * whose return value is finally passed to `hind`. The return value of
+ * `R.around` is the result of this call to `hind`.
+ *
+ * This can be useful for converting some type `a` to be compatible with some
+ * function `(b -> b)` and back again, where an isomorphism exists between
+ * `a` and `b`: `R.around(aToB, bToA, bFn)`.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig (a -> b) -> (c -> d) -> (b -> c) -> a -> d
+ * @param {Function} fore
+ * @param {Function} hind
+ * @param {Function} fn
+ * @param {*}        x
+ * @return {*}
+ * @example
+ *
+ *      var toCharCodes = R.compose(R.map(R.invoker(1, 'charCodeAt')(0)), R.split(''));
+ *      var fromCharCodes = R.apply(String.fromCharCode);
+ *      var strToCharCodes = R.around(toCharCodes, fromCharCodes);
+ *      var printableCharCodes = R.reject(R.either(R.lt(R.__, 32), R.equals(127)));
+ *      var printableStr = strToCharCodes(printableCharCodes);
+ *
+ *      printableStr("foo\nbar\tbaz"); //=> "foobarbaz"
```

Comment:
The name `around` sounds somewhat off to me. 

```js
const f  = R.around(stringToFoo, fooToString);
```

So the returned `f` would have the signature like `f :: (Foo -> Foo) -> Foo`. It allows me to put a function *between* `stringToFoo` and `fooToString`. 

Now I am not suggesting the function should be called `R.between` but calling it `R.around` because the first two arguments will be wrapped around the last is odd to me.

By the way `fore` and `hind` also sound odd to me ;-) Maybe because I am not a native speaker ?